tests/yam/agama/auto.pm

```diff
@@ -13,7 +13,7 @@ use testapi 'assert_screen';
 
 sub run {
     assert_screen('agama-main-page', 120);
-    assert_screen('agama-installing', 60);
+    assert_screen('agama-installing', 240);
```

Comment:
Please check if this timeout really need to be increased once you will create jsonnet format, I'm guessing this is due to the storage that needs more time.
For the future it will be useful if you leave a comment about why you change existing code, it helps to speed up the code review (I'm referring to a comment in the 1st comment of the PR, not in the code). Another option is to use different commits.

Reply:
yes. sometimes it can't catch the screen during that time and fail out. I'd run more tests to verify it. thanks